src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -139,6 +163,16 @@ internal struct CURLMsg
             internal int result;
         }
 
+        // Curl Version Information
+        [StructLayout(LayoutKind.Explicit)]
+        internal struct curl_version_info_data
+        {
+            [FieldOffset(0)]
+            internal int age;
+            [FieldOffset(32)]
+            internal int features;
+        }
```

Comment:
Ack. But I thought this would be costlier - runtime would try to marshal the two pointers and one integer as well. And in case of explicit, it is just a pointer addition. But fair point about arch differences. 
Sending out a PR with your comments.